bin/blueoak-server.js

```diff
@@ -3,27 +3,33 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
-var parseArgs = require('minimist');
+var cli = require('commander');
+var pkg = require('../package.json');
 var server = require('../');
 
-// parse arguments
-var argv = parseArgs(process.argv.slice(2));
-
-// convert mocks from CSV into an array
-var mocks = argv.mocks || argv.m;
-if (mocks) {
-    mocks = mocks.split(',');
-}
+cli = parseOptions();
 
 server.init({
     appDir: process.cwd(),
-    mocks: mocks
+    mockServices: cli.mockServices
```

Comment:
would:
```javascript
mocks: {
  services: cli.mockServices
}
```
work?

Then it can be expanded to support middleware in the future

Reply:
Yep, I like this better.